src/driver-hidpp10.c

```diff
@@ -199,6 +201,14 @@ hidpp10drv_map_button(struct ratbag_device *device,
 		}
 	}
 
+	if (button->action.type == RATBAG_BUTTON_ACTION_TYPE_KEY) {
+		ret = ratbag_button_macro_new_from_keycode(button, button->action.action.key.key, modifiers);
```

Comment:
`rc` as return variable like below

Reply:
We are using `ret` in this function. Using `rc` means creating a new variable, or bringing unrelated changes (renaming `ret` to `rc`) that just make the patch harder to understand.

We should go around the code base converting standardizing the return code variable name.